### DISCUSSION TITLE
Does it works for windows also?

### DISCUSSION THREAD

**Discussion by maor365scores** [Bhupesh-V/ugit#31] [General]
Does it works for windows also?

**Comment by Bhupesh-V** [comment]
> Does it works for windows also?

Hey, yes it does. Although you would have to use the new windows terminal to get it working.
Please read this thread for more info https://github.com/Bhupesh-V/ugit/issues/20#issuecomment-1048234725

Hope this answers your question. Let us know in that same thread if you encounter any issues while setting it up.
